src/Qir/Runtime/lib/QIR/QubitManager.cpp

```diff
@@ -491,7 +457,7 @@ namespace Quantum
 
     void CQubitManager::ReleaseQubitId(QubitIdType id)
     {
-        FailIf(id < 0 || id >= qubitCapacity, "Internal Error: Cannot release an invalid qubit.");
+        FailIf(id >= qubitCapacity, "Internal Error: Cannot release an invalid qubit.");
```

Comment:
Nit? Any details about why the `id < 0 ||` check has been excluded?

Reply:
Whoops! I had removed that check when converting to unsigned type, but then found that wouldn't work with the Qubit Manager design since it needs negative numbers to mark certain qubits. I'll put this one back.